### DISCUSSION TITLE
Outputs

### DISCUSSION THREAD

**Discussion by Hockey8319** [cwendt94/espn-api#354] [Q&A]
I'm a newbie when it come to github and the python environment and I've been struggling getting this to work all day. I'm wondering how much longer I should spend on this, but it all comes down to the following question: What are the outputs of the code? Am I going to have to use the code provided to add more of my code? I'm trying to get more engaged as a commish in to my league this year and I would like to see ESPN projections vs. reality. I would also like to graph the scores as they happen and provide a visual about how a team won (similar to what Yahoo! already does). 

Am I going to get any of that with this? Is there an excel (see newbie) output that i can manipulate or what? 

Pat

**Comment by cwendt94** [comment]
This package basically allows you to get a lot of the data you see on the ESPN website in code and allows the user to use the data however they would like. The [wiki](https://github.com/cwendt94/espn-api/wiki) shows the different functions and classes that hold all of your leagues data.

So you would most likely need to write code using this package to output/format the data however you would like
